deployment/DEPLOY.md

```diff
@@ -1,3 +1,27 @@
 # Deployment
 
-TODO add docu
\ No newline at end of file
+TODO add docu
```

Comment:
is there a ticket for the todo ? :)

(i know it's unrelated to this MR, but would be cool if we could document that eventually)

Reply:
we have #282